filter_plugins/openshift_master.py

```diff
@@ -501,7 +501,36 @@ def validate_pcs_cluster(data, masters=None):
                 valid = False
         return valid
 
+    @staticmethod
+    def certificates_to_synchronize(hostvars):
```

Comment:
I'm now following what you mean by not using the input for the filter plugin... This might be better suited for a lookup plugin instead of a filter plugin.